apps/java-api/src/main/java/org/scoalaonline/api/service/JWTService.java

```diff
@@ -0,0 +1,66 @@
+package org.scoalaonline.api.service;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.JWTVerifier;
+import com.auth0.jwt.algorithms.Algorithm;
+import com.auth0.jwt.interfaces.DecodedJWT;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Contains the logic for managing JWT
+ */
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class JWTService {
+  private Algorithm algorithm = Algorithm.HMAC256("secret".getBytes());
+  private Date access_timer =  new Date(System.currentTimeMillis() + 24 * 60 * 60 * 1000);
+  private Date refresh_timer =  new Date(System.currentTimeMillis() + 24 * 60 * 60 * 1000);
```

Comment:
```suggestion
  private Date accessTimer =  new Date(System.currentTimeMillis() + 24 * 60 * 60 * 1000);
  private Date refreshTimer =  new Date(System.currentTimeMillis() + 24 * 60 * 60 * 1000);
```
Refactor these variables to use *camelCase*.